README.md

```diff
@@ -93,7 +93,7 @@ These are the arguments that apply to all commands:
 
 #### Requirements
```

Comment:
I'd suggest we just remove this Requirements section. No need to maintain the go version, which easily gets outdated, and also it's not a requirement to have ODF installed just to build the tool.

Reply:
Right, this just creates more work for maintaining the readme and people know how to look up go version in go.mod.

@subhamkrai what do you think?